pkg/minishift/clusterup/clusterup.go

```diff
@@ -99,6 +100,9 @@ func ClusterUp(config *ClusterUpConfig, clusterUpParams map[string]string, runne
 		}
 	}
 
+	if glog.V(5) {
+		fmt.Println("-- Running `oc` with: '%s'", strings.Join(cmdArgs, " "))
```

Comment:
@thomasmckay fmt.Println can not print formatted strings. So you need to use fmt.Sprintf i.e. `fmt.Println(fmt.Sprintf("-- Running `oc` with: '%s'", strings.Join(cmdArgs, " ")))`

Reply:
Also I think `fmt.Println(fmt.Sprintf("-- Running oc '%s'", strings.Join(cmdArgs, " ")))` would be better.

Reply:
> Also I think fmt.Println(fmt.Sprintf("-- Running oc '%s'", strings.Join(cmdArgs, " "))) would be better

I would say use `fmt.Printf("-- Running `oc` with: '%s'\n", strings.Join(cmdArgs, " "))` instead calling 2 different functions.

Reply:
`fmt.Printf("%s\n",sth)` is the preferred approach

Reply:
> I would say use fmt.Printf("-- Runningocwith: '%s'\n", strings.Join(cmdArgs, " ")) instead calling 2 different functions.

:+1:  This is fine @thomasmckay

Reply:
switched to fmt.Printf()